Flutter/pdf-viewer/getting-started.md

```diff
@@ -96,4 +96,43 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## Using the controller Property in SfPdfViewer
+
+The [controller](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer/controller.html) property in the SfPdfViewer widget allows you to interact with the PDF document programmatically using the [PdfViewerController](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfViewerController-class.html). This controller provides access to various methods and properties such as [jumpToPage](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfViewerController/jumpToPage.html), [zoomLevel](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfViewerController/zoomLevel.html), [pageCount](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfViewerController/pageCount.html), and annotation management.
+
+he following code explains how to initialize, attach, and dispose the `PdfViewerController`.
```

Comment:
Please cross check the typo errors
<img width="286" height="62" alt="image" src="https://github.com/user-attachments/assets/ac64c404-504d-48e8-8659-82d61bcee55f" />